src/clp_logging/handlers.py

```diff
@@ -364,6 +367,7 @@ def _aggregator(
         timestamp_format: Optional[str],
         timezone: Optional[str],
         timeout: int,
+        zstd_compress: bool,
```

Comment:
Can we hide the compression algorithm for now. I.e. `zstd_compress` -> `enable_compression`